build.sbt

```diff
@@ -21,6 +21,7 @@ lazy val `akka-discovery` = project
   .settings(unidocSettings)
   .settings(
     name := "akka-discovery",
+    organization := "com.lightbend.akka",
```

Comment:
Why only on these two submodules?

Reply:
changing it for artifacts that we have already been publishing for a while might cause confusion with old versions not being properly evicted when they're pulled in as a transitive dependency?

Reply:
> Why only on these two submodules?

Because that's "akka discovery" does not have to live at all with management, could be a core akka thing. Always was intended as such and I did not notice the package was set wrong in build.sbt

> changing it for artifacts that we have already been publishing for a while might

They were first released just yesterday, as 0.7, and that package was a mistake to begin with, even docs pointing out the "com.lightbend.akka" package, so I consider this a bug fix, not a change of package to be honest.

Reply:
i agree it makes sense to make this change for akka-discovery - doing it as well for the other submodules would be more problematic

Reply:
Very confusing that the same project has submodules publishing to different groups IMO.

Reply:
Should discovery be moved into akka then? Makes its quick evolution during 0.x cycle difficult, thus keeping it here and evolving and perhaps splitting off I guess.

Reply:
Nah, It's not that important I guess.